gateway/src/apicast/proxy.lua

```diff
@@ -126,10 +128,37 @@ local function output_debug_headers(service, usage, credentials)
   end
 end
 
+-- Converts a usage to the format expected by the 3scale backend client.
+local function format_usage(usage)
```

Comment:
Maybe it could be a method of `usage.lua`?  Like `self.format`.
It would also facilitate its testing.

Reply:
I'd rather leave the responsibility of dealing with this to the backend client.

The reason is that, otherwise, we'll need to include a similar method on each of our 'model' classes that need to be sent in the params of a request.